mito-ai/src/Extensions/AiChat/ChatMessage/ChatMessage.tsx

```diff
@@ -109,9 +110,12 @@ const ChatMessage: React.FC<IChatMessageProps> = ({
                 } else {
                     return (
                         <div style={{ position: 'relative' }}>
-                            <p onDoubleClick={() => setIsEditing(true)}>
+                            <p key={index} onDoubleClick={() => setIsEditing(true)}>
```

Comment:
I don't think that this key is unique enough. Notice in this video that if I:
1. Add a prompt to the chat taskpane and send the message
2. Then use the explain button to clear the chat and send another prompt
3. The user chat message has both the original user input and the new explain user input

I think we can fix this by making the key `key={index + messagePart}`

https://github.com/user-attachments/assets/79123c50-5c4b-4ee0-a123-68e995a6c38e